grc/core/generator/cpp_top_block.py

```diff
@@ -330,6 +382,11 @@ def make_port_sig(port):
             # Remove the virtual connection
             connections.remove(connection)
 
+        # Remove virtual sink connections too
+        virtual = [c for c in connections if isinstance(c.sink_block, blocks.VirtualSink)]
+        for connection in virtual:
+            connections.remove(connection)
+
```

Comment:
I think we should consider changing this so that it matches the corresponding code in top_block.py:

 https://github.com/gnuradio/gnuradio/blob/590d73242d5755d3b98fc86e2f8f46ae4d9906a8/grc/core/generator/top_block.py#L252-L264

Both work, but it seems cleaner to implement them identically since they are responsible for completing the _exact_ same task